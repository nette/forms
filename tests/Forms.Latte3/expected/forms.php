<?php
%A%
		$this->global->forms->begin($form = $this->global->uiControl['myForm']) /* line %d% */;
		echo $this->global->forms->renderFormBegin(['id' => 'myForm', 'class' => 'ajax']) /* line %d% */;
		echo "\n";
		foreach (['id', 'username', 'select', 'area', 'send'] as $name) /* line %d% */ {
			echo '		';
			echo ($ʟ_label = $this->global->forms->item($name)->getLabel()) /* line %d% */;
			echo '
		';
			echo $this->global->forms->item($name)->getControl()->addAttributes(['title' => 'Hello', 'size' => 10]) /* line %d% */;
			echo '
		';
			echo LR\Filters::escapeHtmlText($this->global->forms->item($name)->getError()) /* line %d% */;
			echo '

		<br>

		';
			echo ($ʟ_label = $this->global->forms->item($form[$name])->getLabel())?->addAttributes(['title' => 'hello'])?->startTag() /* line %d% */;
			echo ' ';
			echo $this->global->forms->item($form[$name])->getControl()->addAttributes(['title' => 'Hello', 'size' => 10]) /* line %d% */;
			echo ' ';
			echo $ʟ_label?->endTag() /* line %d% */;
			echo '
		';
			echo LR\Filters::escapeHtmlText($this->global->forms->item($form[$name])->getError()) /* line %d% */;
			echo "\n";

		}

		echo '
	';
		echo ($ʟ_label = $this->global->forms->item($form['username'])->getLabel()) /* line %d% */;
		echo '

	<LABEL';
		echo ($ʟ_elem = $this->global->forms->item('username')->getLabelPart())->addAttributes(['title' => null])->attributes() /* line %d% */;
		echo ' title=hello>Name</LABEL>
	<input value=val type class="hello"';
		echo ($ʟ_elem = $this->global->forms->item('username')->getControlPart())->addAttributes(['value' => null, 'type' => null, 'class' => null])->attributes() /* line %d% */;
		echo '>

	<label';
		echo ($ʟ_elem = $this->global->forms->item($form['username'])->getLabelPart())->attributes() /* line %d% */;
		echo '></label>
	<label';
		echo ($ʟ_elem = $this->global->forms->item($form['username'])->getLabelPart())->attributes() /* line %d% */;
		echo '>';
		echo $ʟ_elem->getHtml() /* line %d% */;
		echo '</label>
	<input';
		echo ($ʟ_elem = $this->global->forms->item($form['username'])->getControlPart())->attributes() /* line %d% */;
		echo '>

	';
		echo ($ʟ_label = $this->global->forms->item('my')->getLabel()) /* line %d% */;
		echo $this->global->forms->item('my')->getControl() /* line %d% */;
		echo "\n";
		echo $this->global->forms->renderFormEnd() /* line %d% */;
		$this->global->forms->end();

		echo '


';
		$this->global->forms->begin($form = $this->global->uiControl['myForm']) /* line %d% */;
		echo $this->global->forms->renderFormBegin([]) /* line %d% */;
		echo $this->global->forms->renderFormEnd() /* line %d% */;
		$this->global->forms->end();

		echo '

';
		$this->global->forms->begin($form = $this->global->uiControl['myForm']) /* line %d% */;
		echo $this->global->forms->renderFormBegin([]) /* line %d% */;
		echo "\n";
		foreach ($form['sex']->items as $key => $label) /* line %d% */ {
			echo '	';
			echo ($ʟ_label = $this->global->forms->item('sex')->getLabelPart($key))?->startTag() /* line %d% */;
			echo ' ';
			echo $this->global->forms->item('sex')->getControlPart($key) /* line %d% */;
			echo ' ';
			echo LR\Filters::escapeHtmlText($label) /* line %d% */;
			echo $ʟ_label?->endTag() /* line %d% */;
			echo '
	<label';
			echo ($ʟ_elem = $this->global->forms->item('sex')->getLabelPart($key))->addAttributes(['title' => null])->attributes() /* line %d% */;
			echo ' title=hello> <input';
			echo ($ʟ_elem = $this->global->forms->item('sex')->getControlPart($key))->attributes() /* line %d% */;
			echo '> </label>
	<label';
			echo ($ʟ_elem = $this->global->forms->item('sex')->getLabelPart($key))->addAttributes(['title' => null])->attributes() /* line %d% */;
			echo ' title=hello>';
			echo $ʟ_elem->getHtml() /* line %d% */;
			echo '</label>
';

		}

		echo '<label';
		echo ($ʟ_elem = $this->global->forms->item('sex')->getLabelPart())->attributes() /* line %d% */;
		echo '></label>
<label';
		echo ($ʟ_elem = $this->global->forms->item('sex')->getLabelPart())->attributes() /* line %d% */;
		echo '>';
		echo $ʟ_elem->getHtml() /* line %d% */;
		echo '</label>
<label';
		echo ($ʟ_elem = $this->global->forms->item('sex')->getLabelPart())->addAttributes(['title' => null])->attributes() /* line %d% */;
		echo ' title="hello">';
		echo $ʟ_elem->getHtml() /* line %d% */;
		echo '</label>
<input';
		echo ($ʟ_elem = $this->global->forms->item($form['sex'])->getControlPart("{$key}"))->attributes() /* line %d% */;
		echo '>


';
		echo ($ʟ_label = $this->global->forms->item('checkbox')->getLabelPart(''))?->startTag() /* line %d% */;
		echo ' ';
		echo $this->global->forms->item('checkbox')->getControlPart('') /* line %d% */;
		echo ' Label';
		echo $ʟ_label?->endTag() /* line %d% */;
		echo '
<label';
		echo ($ʟ_elem = $this->global->forms->item('checkbox')->getLabelPart(''))->addAttributes(['title' => null])->attributes() /* line %d% */;
		echo ' title=hello> <input';
		echo ($ʟ_elem = $this->global->forms->item('checkbox')->getControlPart(''))->attributes() /* line %d% */;
		echo '> </label>
<label';
		echo ($ʟ_elem = $this->global->forms->item('checkbox')->getLabelPart())->addAttributes(['title' => null])->attributes() /* line %d% */;
		echo ' title=hello> <input';
		echo ($ʟ_elem = $this->global->forms->item('checkbox')->getControlPart())->attributes() /* line %d% */;
		echo '> </label>
<label';
		echo ($ʟ_elem = $this->global->forms->item('checkbox')->getLabelPart(''))->attributes() /* line %d% */;
		echo '>';
		echo $ʟ_elem->getHtml() /* line %d% */;
		echo '</label>
<label';
		echo ($ʟ_elem = $this->global->forms->item('checkbox')->getLabelPart())->addAttributes(['title' => null])->attributes() /* line %d% */;
		echo ' title=hello>';
		echo $ʟ_elem->getHtml() /* line %d% */;
		echo '</label>


';
		foreach ($form['checklist']->items as $key => $label) /* line %d% */ {
			echo '	';
			echo ($ʟ_label = $this->global->forms->item('checklist')->getLabelPart($key))?->startTag() /* line %d% */;
			echo ' ';
			echo $this->global->forms->item('checklist')->getControlPart($key) /* line %d% */;
			echo ' ';
			echo LR\Filters::escapeHtmlText($label) /* line %d% */;
			echo $ʟ_label?->endTag() /* line %d% */;
			echo '
	<label';
			echo ($ʟ_elem = $this->global->forms->item('checklist')->getLabelPart($key))->attributes() /* line %d% */;
			echo '> <input';
			echo ($ʟ_elem = $this->global->forms->item('checklist')->getControlPart($key))->addAttributes(['title' => null])->attributes() /* line %d% */;
			echo ' title=hello> </label>
	<label';
			echo ($ʟ_elem = $this->global->forms->item('checklist')->getLabelPart($key))->attributes() /* line %d% */;
			echo '>';
			echo $ʟ_elem->getHtml() /* line %d% */;
			echo '</label>
';

		}

		echo '<label';
		echo ($ʟ_elem = $this->global->forms->item('checklist')->getLabelPart())->attributes() /* line %d% */;
		echo '></label>
<label';
		echo ($ʟ_elem = $this->global->forms->item('checklist')->getLabelPart())->attributes() /* line %d% */;
		echo '>';
		echo $ʟ_elem->getHtml() /* line %d% */;
		echo '</label>
<label';
		echo ($ʟ_elem = $this->global->forms->item('checklist')->getLabelPart())->addAttributes(['title' => null])->attributes() /* line %d% */;
		echo ' title="hello">';
		echo $ʟ_elem->getHtml() /* line %d% */;
		echo '</label>


';
		if (1) /* line %d% */ {
			$this->global->forms->begin($form = $this->global->uiControl['myForm']) /* line %d% */;
			echo '<form';
			echo $this->global->forms->renderFormBegin(['id' => null, 'class' => null], false) /* line %d% */;
			echo ' id="myForm" class="ajax">
	<input';
			echo ($ʟ_elem = $this->global->forms->item('username')->getControlPart())->attributes() /* line %d% */;
			echo '>
';
			echo $this->global->forms->renderFormEnd(false) /* line %d% */;
			echo '</form>
';
			$this->global->forms->end();
		}
		echo '

';
		$this->global->forms->begin($form = $this->global->uiControl['myForm']) /* line %d% */;
		echo '<form';
		echo $this->global->forms->renderFormBegin(['class' => null], false) /* line %d% */;
		echo ($ʟ_tmp = array_filter(['nclass'])) ? ' class="' . LR\Filters::escapeHtmlAttr(implode(" ", array_unique($ʟ_tmp))) . '"' : "" /* line %d% */;
		echo '>
	<input';
		echo ($ʟ_elem = $this->global->forms->item('username')->getControlPart())->addAttributes(['class' => null])->attributes() /* line %d% */;
		echo ($ʟ_tmp = array_filter(['nclass'])) ? ' class="' . LR\Filters::escapeHtmlAttr(implode(" ", array_unique($ʟ_tmp))) . '"' : "" /* line %d% */;
		echo '>
';
		echo $this->global->forms->renderFormEnd(false) /* line %d% */;
		echo '</form>
';
		$this->global->forms->end();
		echo '

';
		$this->global->forms->begin($form = (is_object($ʟ_tmp = $this->global->uiControl['myForm']) ? $ʟ_tmp : $this->global->uiControl[$ʟ_tmp])) /* line %d% */;
		echo '<FORM';
		echo $this->global->forms->renderFormBegin([], false) /* line %d% */;
		echo '>
	<input';
		echo ($ʟ_elem = $this->global->forms->item('username')->getControlPart())->attributes() /* line %d% */;
		echo '>
';
		echo $this->global->forms->renderFormEnd(false) /* line %d% */;
		echo '</FORM>
';
		$this->global->forms->end();
		echo '

<select';
		echo ($ʟ_elem = $this->global->forms->item('select')->getControlPart())->attributes() /* line %d% */;
		echo '>';
		echo $ʟ_elem->getHtml() /* line %d% */;
		echo '</select>


<textarea';
		echo ($ʟ_elem = $this->global->forms->item('area')->getControlPart())->addAttributes(['title' => null])->attributes() /* line %d% */;
		echo ' title="';
		echo LR\Filters::escapeHtmlAttr(10) /* line %d% */;
		echo '">';
		echo $ʟ_elem->getHtml() /* line %d% */;
		echo '</textarea>


<select';
		echo ($ʟ_elem = $this->global->forms->item('select')->getControlPart())->attributes() /* line %d% */;
		echo '>';
		echo $ʟ_elem->getHtml() /* line %d% */;
		echo '</select>
';
		echo $this->global->forms->renderFormEnd() /* line %d% */;
		$this->global->forms->end();

		echo '


';
		$this->global->forms->begin($form = $this->global->uiControl['myForm']) /* line %d% */;
		echo '
<label';
		echo ($ʟ_elem = $this->global->forms->item('sex')->getLabelPart())->attributes() /* line %d% */;
		echo '>';
		echo $ʟ_elem->getHtml() /* line %d% */;
		echo '</label>
<input';
		echo ($ʟ_elem = $this->global->forms->item('username')->getControlPart())->attributes() /* line %d% */;
		echo '>
';
		/* line %d% */;
		$this->global->forms->end();
%A%
