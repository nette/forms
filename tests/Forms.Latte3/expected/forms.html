<form action="" method="post" class="ajax" id="myForm">

		<input type="hidden" name="id" value="" title="Hello" size="10">


		<br>

		 <input type="hidden" name="id" value="" title="Hello" size="10">

		<label for="frm-username">Username:</label>
		<input type="text" name="username" size="10" class="control-class" id="frm-username" data-nette-error title="Hello">
		error

		<br>

		<label for="frm-username" title="hello"> <input type="text" name="username" size="10" class="control-class" id="frm-username" data-nette-error title="Hello"> </label>
		error
		<label for="frm-select"></label>
		<select name="select" size="10" id="frm-select" title="Hello"><option value="m">male</option><option value="f">female</option></select>


		<br>

		<label for="frm-select" title="hello"> <select name="select" size="10" id="frm-select" title="Hello"><option value="m">male</option><option value="f">female</option></select> </label>

		<label for="frm-area"></label>
		<textarea name="area" id="frm-area" title="Hello" size="10">one&lt;two</textarea>


		<br>

		<label for="frm-area" title="hello"> <textarea name="area" id="frm-area" title="Hello" size="10">one&lt;two</textarea> </label>


		<input type="submit" name="send" value="Sign in" title="Hello" size="10">


		<br>

		 <input type="submit" name="send" value="Sign in" title="Hello" size="10">


	<label for="frm-username">Username:</label>

	<LABEL for="frm-username" title=hello>Name</LABEL>
	<input value=val type class="hello" name="username" id="frm-username" data-nette-error>

	<label for="frm-username"></label>
	<label for="frm-username">Username:</label>
	<input type="text" name="username" class="control-class" id="frm-username" data-nette-error>

	<label>My</label><input name=My>
</form>



<form action="" method="post" class="form-class"><input type="hidden" name="id" value=""></form>


<form action="" method="post" class="form-class">
	<label for="frm-sex-m"> <input type="radio" name="sex" id="frm-sex-m" value="m"> male</label>
	<label for="frm-sex-m" title=hello> <input type="radio" name="sex" id="frm-sex-m" value="m"> </label>
	<label for="frm-sex-m" title=hello>male</label>
	<label for="frm-sex-f"> <input type="radio" name="sex" id="frm-sex-f" value="f"> female</label>
	<label for="frm-sex-f" title=hello> <input type="radio" name="sex" id="frm-sex-f" value="f"> </label>
	<label for="frm-sex-f" title=hello>female</label>
<label></label>
<label>Sex:</label>
<label title="hello">Sex:</label>
<input type="radio" name="sex" id="frm-sex-f" value="f">


<label for="frm-checkbox"> <input type="checkbox" name="checkbox" id="frm-checkbox"> Label</label>
<label for="frm-checkbox" title=hello> <input type="checkbox" name="checkbox" id="frm-checkbox"> </label>
<label for="frm-checkbox" title=hello> <input type="checkbox" name="checkbox" id="frm-checkbox"> </label>
<label for="frm-checkbox">Checkbox</label>
<label for="frm-checkbox" title=hello>Checkbox</label>


	<label for="frm-checklist-m"> <input type="checkbox" name="checklist[]" id="frm-checklist-m" value="m"> male</label>
	<label for="frm-checklist-m"> <input type="checkbox" name="checklist[]" id="frm-checklist-m" value="m" title=hello> </label>
	<label for="frm-checklist-m">male</label>
	<label for="frm-checklist-f"> <input type="checkbox" name="checklist[]" id="frm-checklist-f" value="f"> female</label>
	<label for="frm-checklist-f"> <input type="checkbox" name="checklist[]" id="frm-checklist-f" value="f" title=hello> </label>
	<label for="frm-checklist-f">female</label>
<label></label>
<label>CheckboxList:</label>
<label title="hello">CheckboxList:</label>


<form action="" method="post" id="myForm" class="ajax">
	<input type="text" name="username" class="control-class" id="frm-username" data-nette-error>
<input type="hidden" name="id" value=""></form>


<form action="" method="post" class="nclass">
	<input type="text" name="username" id="frm-username" data-nette-error class="nclass">
<input type="hidden" name="id" value=""></form>


<FORM action="" method="post" class="form-class">
	<input type="text" name="username" class="control-class" id="frm-username" data-nette-error>
<input type="hidden" name="id" value=""></FORM>


<select name="select" id="frm-select"><option value="m">male</option><option value="f">female</option></select>


<textarea name="area" id="frm-area" title="10">one&lt;two</textarea>


<select name="select" id="frm-select"><option value="m">male</option><option value="f">female</option></select>
</form>
